scripts/prebuild.js

```diff
@@ -0,0 +1,56 @@
+const { spawnSync } = require('child_process');
+const path = require('path');
+
+const pkg = require(path.resolve('package.json'));
+
+const regex = new RegExp('^\\d+\.\\d+\.\\d+$');
+
+function spawnPrebuild(args) {
+  let command = 'prebuild';
```

Comment:
Is importing the module and running like that an option? Doing this will prevent running via `./scripts/prebuild.js` since it's not on the `$PATH`.

Reply:
This script should be run via npm which will set $PATH correctly.

Reply:
@amejia1 is it possible to `require('prebuild')` and use it like that? That way the script should work both ways. Otherwise running 'node_modules/.bin/prebuild' would be preferable to this so it can be run without npm.